ag_data/fixtures/sample.json

```diff
@@ -0,0 +1,128 @@
+[
```

Comment:
This is minor, but is there a way to only load sensor types into the db when initiating with the new design?

Reply:
Currently not but we can separate `sample.json` into two files containing sensor types and other models so that we could selectively load it.

Reply:
Yeah, that can do.